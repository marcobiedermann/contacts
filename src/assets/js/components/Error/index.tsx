import React, { FC } from 'react';
import styles from './style.module.css';

const Error: FC = (props) => <div className={styles.error} {...props} />;

export default Error;
