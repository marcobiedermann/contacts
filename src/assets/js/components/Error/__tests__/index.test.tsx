import { render } from '@testing-library/react';
import React from 'react';
import Error from '..';

describe('error component', () => {
  it('should render correctly', () => {
    expect.assertions(1);

    const { asFragment } = render(<Error>Error Message</Error>);

    expect(asFragment()).toMatchSnapshot();
  });
});
