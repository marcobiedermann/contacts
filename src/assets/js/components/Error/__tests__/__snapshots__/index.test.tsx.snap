// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="error"
  >
    Error Message
  </div>
</DocumentFragment>
`;
